projects/04_cube_xr/build.gradle

```diff
@@ -0,0 +1,11 @@
+task copyTask {
```

Comment:
Also re-add the simple 04_cube (existed before)

Reply:
oops, re-added, thanks!